src/core/client/base.js

```diff
@@ -81,9 +81,6 @@ export default async function createClient(
       // When `forceRefresh` is `true`, client side navigation is disabled and
       // all links will trigger a full page reload. This is what we want when
       // the server has loaded an anoynmous page.
-      //
-      // Note: we check the presence of the `site` state because the `disco`
-      // app uses the same code but does not use this state.
       forceRefresh:
         initialState && initialState.site
           ? initialState.site.loadedPageIsAnonymous
```

Comment:
seems like we won't need the check on `.site` then?

Reply:
I added a note to: https://github.com/mozilla/addons-frontend/issues/9805